src/main/java/fr/xephi/authme/settings/properties/SecuritySettings.java

```diff
@@ -132,6 +132,15 @@
     public static final Property<Integer> EMAIL_RECOVERY_COOLDOWN_SECONDS =
         newProperty("Security.emailRecovery.cooldown", 60);
 
+    @Comment({
+        "The maill showed using /email show will be partially hidden",
+        "E.g. (if enabled)",
+        " original email: myemail@domain.com",
+        " hidden email: mye***@***in.com"
+    })
+    public static final Property<Boolean> EMAIL_PRIVACY =
+        newProperty("Security.privacy.email", false);
```

Comment:
I was going to suggest to name it kind of like a question since it's a boolean, but then I started thinking of propositions (`MASK_EMAIL`, `USE_EMAIL_MASKING` or `HIDE_FULL_EMAIL`) and they don't really seem much better xP